vagrant/provision.sh

```diff
@@ -79,7 +79,11 @@ done
 # install requirements for ci/build.sh
 snap install ruby --channel=2.6/stable --classic
 gem install bundler
-apt install -y gcc g++ libsnappy-dev shellcheck
+apt install -y gcc g++ libsnappy-dev
+
+wget -qO- https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz | tar -xJf -
```

Comment:
Can we extract the version into VARIABLE like we do for `HELM_2_VERSION` and `HELM_3_VERSION`?